website/source/docs/providers/aws/d/db_snapshot.html.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: "aws"
+page_title: "AWS: aws_db_snapshot"
+sidebar_current: "docs-aws-datasource-db-snapshot"
+description: |-
+  Get information on a DB Snapshot.
+---
+
+# aws\_db\_snapshot
+
+Use this data source to get information about a DB Snapshot for use when provisioning DB instances
+
+## Example Usage
+
+```
+data "aws_db_snapshot" "ebs_volume" {
```

Comment:
nit: `ebs_volume` sounds like a weird data source name when referring to a db snapshot 😄